ehrql/dummy_data_nextgen/generator.py

```diff
@@ -101,9 +101,7 @@ def __init__(
                 lhs=variable_definitions["population"],
                 rhs=self.configuration.additional_population_constraint._qm_node,
             )
-        self.population_size = (
-            configuration.population_size or DummyDataConfig.population_size
-        )
+        self.population_size = configuration.population_size
```

Comment:
It is. I'll squash the commits to make it less confusing. I got the way we were configuring population size for measures wrong and ended up having to go back to something closer to the original version, which meant that this could no longer be None and the default was irrelevant.